test/InputFilter/RestService/PatchInputFilterTest.php

```diff
@@ -24,82 +24,18 @@ public function getInputFilter(): PatchInputFilter
         ]);
     }
 
-    /** @psalm-return array<string, array{0: array<string, mixed>}> */
+    /**
+     * @psalm-return array<string, array{
+     *     string|integer
+     * }>
+     */
```

Comment:
```suggestion
    /** @psalm-return array<string, array{0: string|int}> */
```
This is more precise, and will give more information to Psalm.

Reply:
@weierophinney lack of experience with Psalm here. Thanks!